src/components/reusables/reuseableInput.js

```diff
@@ -0,0 +1,32 @@
+// react libraries
+import React from 'react';
+
+// third-party libraries
+import PropTypes from 'prop-types';
+
+/**
+ * @param {object} props
+ * @desc reuseableInput
+ * @return {object} input
+*/
+const reuseableInput = props => (
```

Comment:
#### Critical
I thought we discussed on why this should be in `PascalCase` instead of `camelCase`?